build/Makefile

```diff
@@ -215,7 +215,7 @@ push-release-chart: $(ensure-build-image)
 	CHART_DIR=install/.helm-$(RELEASE_VERSION)/agones $(MAKE) push-chart
 
 #build all the sdks
-build-sdks: build-sdk-cpp
```

Comment:
@dsazonoff should this have been removed? Should we build and/or test this SDK on each run, in case something fails?

Reply:
I just removed c++ related part. Probably yes, we should remove this step.

Reply:
I guess my point is - shouldn't building the SDK be part of the tests, or ensure there isn't some error in it?